docs/studios/unity/unity-sdk/csharp/utilities.mdx

```diff
@@ -0,0 +1,300 @@
+---
+sidebar_label: 'Utilities for C# Bots'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Utilities for C# Bots
+
+## RGGizmos
+
+`RGGizmos` is a utility for drawing specific Gizmos and debug information within your Unity scene. It is
+primarily a wrapper around Unity's built-in [Gizmos feature](https://docs.unity3d.com/ScriptReference/Gizmos.html),
+but it also provides some additional functionality around text drawing and rendering Gizmos from the bot threads.
+The RGGizmos library currently supports three main features:
+
+* Drawing lines betweens entities and positions
+* Drawing speheres at entities and positions
+* Rendering billboard text above an entity 
+
+**Skip to the end of the RGGizmos section to see a video of these in action.**
+
+All methods below can be accessed via the `RGGizmos` variable in your `ProcessTick(RG rgObject)` method.
```

Comment:
I feel like this sentence needs some wording qualification that `ProcessTick` is within your bot code.

Reply:
Fixed! Also linked to our only page that talks about ProcessTick until we add a proper reference.